package.json

```diff
@@ -6,10 +6,12 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
-    "dev": "nodemon --legacy-watch --exec node --loader ts-node/esm src/app.ts",
+    "dev": "nodemon --legacy-watch --exec node --loader ts-node/esm src/server.ts",
```

Comment:
- add prettier as formatter 
- run prettier in precommit
- add aliases for linting & formatting